src/lib/arrays.js

```diff
@@ -0,0 +1,107 @@
+const getNthElement = (index, array) => {
+  if (index < array.length) {
+    return array[index];
+  }
+  return array[index - array.length];
+};
+
+const arrayToCSVString = array => {
+//  console.log(array.toString());
+  return array.toString();
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+function bob(bsr) {
+  return bsr;
+}
+
+const addToArray2 = (element, array) => {
+  const newArray = array.map(bob);
+  newArray.push(element);
+  console.log(newArray);
+  return newArray;
+};
+
+
+const removeNthElement = (index, array) => {
+if (index > -1) {
+  array.splice(index, 1);
+}
+return(array);
+}
+
+const numbersToStrings = numbers => {
+  const arrayToString = numbers.toString();
+  return arrayToString.split(',');
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(words => words.toUpperCase());
+};
+
+const reverseWordsInArray = strings => {
+  const reverse = strings.map(item => item.split('').reverse().join(''));
+  //console.log(reverse);
+  return reverse;
+};
+
+const onlyEven = numbers => {
+  return evenNum = numbers.filter((num) => num % 2 === 0);
+};
+
+const removeNthElement2 = (index, array) => {
+  const copy = array.map(bob);
+ // console.log(copy);
```

Comment:
Console logs should be deleted, the helper function could be based inside the removeNthElement2 (does not have to be). 
copy.splice(index, 1); is assigned to the variable words, but words is not used anywhere, The function reads from top to bottom, therefor you dont have to assign it to a variable for the method to be applied, You function first will create a copy of the array, then splice the copy and then return the copy.